src/apps/annotation-text/TextAnnotationDesktop.tsx

```diff
@@ -336,7 +336,7 @@ export const TextAnnotationDesktop = (props: TextAnnotationProps) => {
 
       <Toast
         content={toast}
-        onOpenChange={(open) => !open && setToast(null)}
+        onOpenChange={(open) => !open && setToast(undefined)}
```

Comment:
Likewise, showed up as a type error in my IDE (due to the way the toast arg is defined).